tutorial/Lab2/README.md

```diff
@@ -0,0 +1,151 @@
+# Lab 2. I need to change but none of the hassle
+
+In [Lab 1](../Lab1/README.md) we installed the app. You are probably now thinking, I am done. In most circumstances that would be correct but what about any updates or improvements to the chart. How do you update your running app to pick up these changes? 
+
+In this lab we are going to look at how to update our running app when the chart has been changed. To be able to demonstrate this we are going to make changes to the original `guestbook` chart as follows: 
+* Remove the Redis slaves and therefore just use the in-memory DB
+* Change from `LoadBalancer` type to `NodePort` type
+
+It is slightly contrived but the end goal here is to show how to update using Kubernetes directly and Helm. So, how easy is it to do this? 
+
+# Update the application using `kubectl`
+
+In this part of the lab we will update the previously deployed application `guestbook`, using Kubernetes directly.
+
+1. This is an optional step in that it is not necessary to update your running app. The reason for doing this step is "house keeping" - you want to have the correct files for the current configuration that you have deployed. This avoids making mistakes if you have future updates or even rollbacks. In this updated configuration, we remove the Redis slaves. That is why you should move/archive or remove the Redis slave files to keep abreast of the update:
+
+   ```
+   redis-slave-service.yaml
+   redis-slave-deployment.yaml
+   ```
+
+2. Delete the Redis slave service and pods:
+
+   ```console
+   $ kubectl delete svc redis-slave --namespace default
+   service "redis-slave" deleted
+   $ kubectl delete deployment redis-slave --namespace default
+   deployment.extensions "redis-slave" deleted
+   ```
+
+3. Update the guestbook service from `LoadBalancer` to `NodePort` type:
+
+   ```$ sed -i.bak 's/LoadBalancer/NodePort/g' guestbook-service.yaml```
+
+   Note: Like in Step 1., you may want to archive before making the changes..
+   
+4. Delete the guestbook service:
+    
+    ```$ kubectl delete svc guestbook --namespace default```
+    
+5. Recreate the service with `NodePort` type:
```

Comment:
`Recreate` should be `Re-create` or `Create the service again with...`

Reply:
done